source/rst/markov_jump_lq.rst

```diff
@@ -158,7 +158,7 @@ with
 
 .. math::
 
-  r(x_t, s_t, u_t) = -( x_t' R_{s_t} x_t + u_t' Q_{s_t} u_t + 2 u_t' W_{s_t} x_t)
+  r(x_t, s_t, u_t) = x_t' R_{s_t} x_t + u_t' Q_{s_t} u_t + 2 u_t' W_{s_t} x_t
```

Comment:
Thank @jstac and @spspitze . 

In my opinion, the change in this PR makes sense
- i.e., $r(x_t, s_t, u_t)$ should be $x_t' R_{s_t} x_t + u_t' Q_{s_t} u_t + 2 u_t' W_{s_t} x_t$, instead of $-( x_t' R_{s_t} x_t + u_t' Q_{s_t} u_t + 2 u_t' W_{s_t} x_t)$

This is because:
- in the section before this change, i.e., in [Review-of-useful-LQ-dynamic-programming-formulas](https://python-advanced.quantecon.org/markov_jump_lq.html#Review-of-useful-LQ-dynamic-programming-formulas), we define the loss function as $r(x_t, u_t) = x_t' R x_t + u_t' Q u_t + 2 u_t' W x_t$
- $r(x_t, s_t, u_t)$ is obtained just by letting matrices $R, Q, W$ depend on the state space of a Finite Markov chain.

Furthermore, the $N$ interrelated Bellman equations satisfied by the optimal value functions $- x' P_i x - \rho_i$ for $i = 1, \ldots, n$ should also be corrected as:
- $-x' P_i x - \rho_i & = \max_u - \left[ x'R_i x + u' Q_i u + 2 u' W_i x - \beta \sum_j \Pi_{ij}E ((A_i x + B_i u + C_i w)' P_j (A_i x + B_i u + C_i w) x + \rho_j) \right]$ -->> $-x' P_i x - \rho_i & = \max_u - \left[ x'R_i x + u' Q_i u + 2 u' W_i x + \beta \sum_j \Pi_{ij}E ((A_i x + B_i u + C_i w)' P_j (A_i x + B_i u + C_i w) x + \rho_j) \right]$

Do you think it is a good idea?